### ISSUE TITLE
Shift from Python 2.7 to Python 3

### ISSUE THREAD

**Issue by nadzhou** [duolinwang/MusiteDeep#8]
The code is valuable we will need to shift it to Python 3 to make it compatible with the latest libraries in tensorflow.

**Comment by duolinwang** [comment]
Please try the version MusiteDeep_Keras2.0 it is compatible with python3. This version also compatible with python 3 just change some print xxx into print(xxx) it should work.

**Comment by nadzhou** [comment]
@duolinwang yeah. I'm thinking of doing it myself. there are some library differences.

**Comment by prihoda** [comment]
Hi @nadzhou and @duolinwang. Is there a version that currently supports Python 3? I see that code in the `MusiteDeep_Keras2.0` directory in this repository still contains python 2 print statements. Thanks!

**Comment by duolinwang** [comment]
It actually works for python 3 if the python 2 print statements are changed into python 3 style. I did experiments using python 3 in other systems, just changed the print statements. I just forgot to update the codes.

Best,
Duolin

在 2021年2月24日，10:01，David Příhoda <notifications@github.com> 写道：

﻿
WARNING: This message has originated from an External Source. This may be a phishing expedition that can result in unauthorized access to our IT System. Please use proper judgment and caution when opening attachments, clicking links, or responding to this email.

Hi @nadzhou<https://nam02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnadzhou&data=04%7C01%7C%7Ca7e8c238664e468d540f08d8d8dd6fe8%7Ce3fefdbef7e9401ba51a355e01b05a89%7C0%7C0%7C637497792860705788%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=PiyvFHfCP8PK6S0CInlUYMM5KpMciJ3JUCWxo3VYry8%3D&reserved=0> and @duolinwang<https://nam02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fduolinwang&data=04%7C01%7C%7Ca7e8c238664e468d540f08d8d8dd6fe8%7Ce3fefdbef7e9401ba51a355e01b05a89%7C0%7C0%7C637497792860705788%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=TC3tYMHG2KmkV%2FHKIGqf3JHBzS3y%2B0RJ6i9dd%2BzwXxw%3D&reserved=0>. Is there a version that currently supports Python 3? I see that code in the MusiteDeep_Keras2.0 directory in this repository still contains python 2 print statements. Thanks!
